osmosis-1/osmosis-1.assetlist.json

```diff
@@ -949,6 +949,31 @@
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dig.png"
       }
     },
+    {
+      "description": "Rowan Token (ROWAN) is the Sifchain Network's native utility token, used as the primary means to govern, provide liquidity, secure the blockchain, incentivize participants, and provide a default mechanism to store and exchange value.",
+      "denom_units": [
+        {
+          "denom": "ibc/8318FD63C42203D16DDCAF49FE10E8590669B3219A3E87676AC9DA50722687FB",
+          "exponent": 0,
+          "aliases": [
+            "rowan"
```

Comment:
@sunnya97 is this going to cause any issues in the osmosis frontend due to rowan's lack of an SI prefix?

Reply:
hrmm, not sure if the alias can be the same as the denom. Sunny / @IbrarMakaveli / @jonator would know better though

Reply:
Yeah, should just remove this

Reply:
👍